cfgov/jinja2/v1/owning-a-home/_templates/no_js.html

```diff
@@ -1,4 +1,16 @@
-<div class="js-error-warning">
-    <h2>You must have JavaScript to use the tools on this page.</h2>
-    To improve your experience, please <a href="https://www.enable-javascript.com/" class="go-link">enable JavaScript in your browser</a>
+<div class="m-notification
+            m-notification__error
+            m-notification__no-js">
+    <span class="m-notification_icon cf-icon"></span>
+    <div class="m-notification_content">
+        <div class="h4 m-notification_message">
+            You must have JavaScript to use the tools on this page.
+        </div>
+        <p class="h4 m-notification_explanation">
+            To improve your experience, please
+            <a href="https://www.enable-javascript.com/" class="go-link">
+                enable JavaScript in your browser
```

Comment:
Looks like this needs a period at the end?

![screen shot 2018-04-24 at 11 27 39 am](https://user-images.githubusercontent.com/704760/39197451-7b076f20-47b2-11e8-9d98-f355a2d7cefa.png)

Reply:
We should hide the whole tool when JS is off, but that's for another time.

Reply:
I set the opacity to .1 for the rate checker in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/805de5edd2f3f85ba299ecef445c033fc1e117a9.

Reply:
Can we not include the text below it? I would just do the tool. Like you can still interact with the feedback form in a semi-opaque form. If that's too much to deal with here I'd keep it how it was before and address hiding it in another PR.

Reply:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/4047/commits/0731f1427922b375e72a51102fa5295b5baf323a.